tests/consensus_tests/test_order_by.py

```
@@ -8,7 +8,7 @@
 
 
 def test_order_by_from_remote_shard(tmp_path, every_test):
-    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, num_peers=2, port_seed=10000)
+    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, num_peers=2)
```

Ruff (0.15.10)

[warning] Unpacked variable `peer_dirs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)